basil.js

```diff
@@ -2223,61 +2223,191 @@ pub.bleeds = function(top, right, bottom, left) {
 
 
 /**
- * Prints out all properties and values off an object in a recursive manner to the console. Useful for inspecting (or debugging) nested variable. the default value for the recursion is maxlevel = 2.
+ * @description Inspects a given object or any other data item and prints the result to the console. This is useful for inspecting or debugging any kind of variable or data item. The optional settings object allows to control the function's output. The following parameters can be set in the settings object:
+ * <code>showProps<code>: Show or hide properties. Default: <code>true<code>
+ * <code>showValues<code>: Show or hide values. Default: <code>true<code>
+ * <code>showMethods<code>: Show or hide methods. Default: <code>false<code>
+ * <code>maxLevel<code>: Chooses how many levels of properties should be inspected recursively. Default: <code>1<code>
+ * <code>propList<code>: Allows to pass an array of property names to show. If propList is not set all properties will be shown. Default: <code>[]<code> (no propList)
+ * If no settings object is set, the default values will be used.
  *
  * @cat Output
  * @method inspect
- * @param  {Object} obj The Object to be inspected.
- * @param  {Number} maxlevel Recursion limit, default maxlevel = 2.
+ * @param  {Object} obj An object or any other data item to be inspected.
+ * @param  {Object} settings A settings object to control the function's behavior.
+ *
+ * @example
+ * // inspecting a string
```

Comment:
ups. The review should have been done on the `environment.js` not on the bundle.